src/ppx_tools/dumpast.mli

```diff
@@ -0,0 +1,9 @@
+val transform : string -> Pp_path.kind -> Jsonoo.t
```

Comment:
This mli needs a top level ocamldoc comment. ocamldoc comments for the individual functions would be welcome as well.